New plugin - basemap switch control

Urban96 opened the pull request:
New plugin featuring a basemap control with an integrated synchronized miniature map

Falke-Design reviewed (changes requested):
Please make the description more clear what the plugin does. 
_... Clicking on the minimap changes the base layer ..._

Also I strongly suggest you to fix following:
1. On zoom level 0 the minimap doesn't work anymore
2. Add [disableClickPorpagation](https://leafletjs.com/reference.html#domevent-disableclickpropagation) to the minimap. This prevents moving the map when the user double clicks on the minimap
3. Use the latest Leaflet version (1.9.4) in your example

Urban96 commented:
Hello,

Thank you for the feedback! I’ve made the necessary updates: clarified the description, fixed the zoom level 0 issue, added disableClickPropagation, and updated to Leaflet 1.9.4.

Falke-Design commented on docs/_plugins/layer-switching-controls/l-basemapcontrol.md:
```diff
@@ -0,0 +1,12 @@
+compatible-v1: true
+---
+
+Basemap Control with Synchronized Minimap
```
Please change this describtion

Urban96 replied:
Done. I hope this description better captures what the plugin does.

Falke-Design commented on docs/_plugins/layer-switching-controls/l-basemapcontrol.md:
```diff
@@ -0,0 +1,12 @@
+compatible-v1: true
+---
+
+Leaflet plugin that adds a control for switching between multiple basemap layers. It features a synchronized minimap preview to show the next layer in the sequence.
```
@Urban96 I would suggest something like this, because for me it was at the beginning not clear that I need to click on the minimap to switch.
But if you want to keep your description, it is fine for me and I will merge it.

```suggestion
Leaflet plugin that creates a synchronized minimap preview to switch betweeen multiple basemap layers. With a click on the minimap the next basemap layer in the sequence is applied.
```

Urban96 replied:
Sure, it's updated! Thanks
